exporter/collector/metricsexporter.go

```diff
@@ -126,22 +158,28 @@ func NewGoogleCloudMetricsExporter(
 		obs:    selfObservability{log: log},
 		mapper: metricMapper{cfg},
 		// We create a buffered channel for metric descriptors.
-		// MetricDescritpors are asycnhronously sent and optimistic.
+		// MetricDescritpors are asychronously sent and optimistic.
 		// We only get Unit/Description/Display name from them, so it's ok
 		// to drop / conserve resources for sending timeseries.
-		mds:     make(chan *metricpb.MetricDescriptor, cfg.MetricConfig.CreateMetricDescriptorBufferSize),
-		mdsDone: make(chan struct{}),
+		metricDescriptorC: make(chan *metricpb.MetricDescriptor, cfg.MetricConfig.CreateMetricDescriptorBufferSize),
+		mdCache:           make(map[string]*metricpb.MetricDescriptor),
+		timeSeriesC:       make(chan *monitoringpb.TimeSeries),
+		shutdownC:         make(chan struct{}, 1),
```

Comment:
nit: I don't think this needs to be a buffered channel

Reply:
Done